src/Period.php

```diff
@@ -517,7 +532,7 @@ protected function createDateInterval(int $precision): DateInterval
             Precision::HOUR => 'PT1H',
             Precision::DAY => 'P1D',
             Precision::MONTH => 'P1M',
-            Precision::YEAR => 'P1Y',
```

Comment:
Could you undo these styling changes?

Reply:
Rolled back last commit to undo changes.